src/sentry/models/file.py

```diff
@@ -274,17 +309,38 @@ def seek(self, pos):
     def tell(self):
         if self.closed:
             raise ValueError('I/O operation on closed file')
+        if self.prefetched:
+            return self._curfile.tell()
+        if self._curfile is None:
+            return self.size
         return self._curidx.offset + self._curfile.tell()
 
-    def read(self, bytes=4096):
+    def read(self, n=-1):
```

Comment:
Isn't this pretty significantly changing behavior here? Since now every `.read()` will read the full thing instead of a chunk? I get that this is the correct API behavior of a file like object, but I want to make sure we don't regress.

Reply:
@mattrobenolt It was never called that way.